tools/update_schema

```diff
@@ -67,7 +72,16 @@ def _get_output_path(version: str) -> Path:
 
 def _load_json(filepath: Path) -> dict[str, Any]:
     with open(filepath, encoding="utf-8") as f:
-        return json.load(f)
+        try:
+            return json.load(f)
+        except json.JSONDecodeError as json_decode_error:
+            print(
+                f"{FAILURE} Parsing existing json schema failed: The json is malformed. "
+                "If you know why, re-run the command with argument '--force' "
+                "to overwrite with the new schema.\n"
+                f"{FAILURE} Json parsing error: '{json_decode_error.msg}.'"
+            )
+            sys.exit() 
```

Comment:
```suggestion
            sys.exit(1) 
```
I think it'd be good to indicate an error status with the exit code

Reply:
Good point. Will add that